SWIG/creditdefaultswap.i

```diff
@@ -172,5 +174,24 @@ class IntegralCdsEnginePtr : public boost::shared_ptr<PricingEngine> {
     }
 };
 
+%rename(IsdaCdsEngine) IsdaCdsEnginePtr;
+class IsdaCdsEnginePtr : public boost::shared_ptr<PricingEngine> {
+  public:
+    %extend {
+        IsdaCdsEnginePtr(
+            const Handle<DefaultProbabilityTermStructure> &probability,
+            Real recoveryRate,
+            const Handle<YieldTermStructure> &discountCurve,
+            bool includeSettlementDateFlows = false,
+            const IsdaCdsEngine::NumericalFix numericalFix = IsdaCdsEngine::Taylor,
```

Comment:
The inner enumerations like `NumericalFix` and `AccrualBias` might need to be exposed.  See `VanillaSwap` in `swap.i` for an example.

Reply:
Sorry it took awhile for me to get back to this.  I've made an attempt at exporting the inner enums now.

Reply:
Thanks.

Reply:
I also added the enums as static variables of the proxy class (they wouldn't be exported otherwise).